CIP-1904/README.md

```diff
@@ -0,0 +1,250 @@
+Usage of CIDs allow us to represent the hash in a self-describing way that is not ambiguous.
+The subtype `st` may limit the types of CIDs used in order to not require 3rd party verification applications to support too many hash algorithms.
+
+The CID itself, which is embedded in the metadata of the transaction is used as an identifier for the batch and hence we recommend using this as a unique identifier to resolve the off-chain data.
+
+### Transaction types
+Currently, there are 3 different types of verifiable supply chain records supported:
+- Supply Chain Management data,
+- Certificate of Conformity issue,
+- Certificate of Conformity revoke
+
+The [CDDL schema](./schema.cddl) covers all transaction types.
+
+#### Supply Chain Management data
+Supply Chain Management (SCM) data is associated with a good or product as a result of production, manufacturing and transport processes throughout its lifecycle.
+This data may be shared between all supply chain stakeholders as well as delivered to end customers.
+A producer is responsible to collect the SCM data associated with its product/good and insert it into the supply chain ecosystem in a defined manner specific to the use-case.
+
+We strongly recommend that the off-chain SCM schema follows an industry standard relevant to the particular use-case.
+For example, the Georgian Wine use-case from the Cardano Foundation with the National Wine Agency of Georgia uses an extended version of the [OIV Standard](https://www.oiv.int/standards/international-standard-for-the-labelling-of-wines) from the [International Organisation of Vine and Wine](https://www.oiv.int).
+Using such standards brings credibility to the transactions and helps foster enterprise adoption.
+
+##### Off-chain format
+Each verifiable record comes from a specific producer by `producerID`.
+An SCM batch can contain records from multiple producers, hence the off-chain data is a JSON map of `producerID` to an array of JSON objects.
+The structure of the JSON objects within the array is dependant on the subtype `st`.
+
+The `producerID` is a unique identifier for a producer in the supply chain ecosystem of the given subtype.
+We recommend short IDs if possible to save on space and costs.
+
+The following example contains 3 verifiable records from producer `a2gh` and producer `p33l`, where `{...}` represents an arbitrary JSON object for a given verifiable record:
+```
+{
+  a2gh: [{...}, {...}, {...}],
+  p33l: [{...}, {...}]
+}
+```
+
+##### On-chain format
+The following field labels are used within the on-chain metadata:
+
+|**Field**|**Description**|**Type**|**Set Value**|
+|---|---|---|---|
+|`t` for type|The type defines the structure of the data.|string|scm|
+|`st` for subtype|This **optional** field can be used to specify/describe a use-case. Please see the Subtypes section for more information.|string||
+|`v` for version|The version of the specification.|string|1|
```

Comment:
This really is a string and not an int - used to be `v1` and never converted it to a uint.

We're already live so not much we can do - but a string is fine I think - at least leaves us open to go something like "1" -> "1.1" in case the change is minor or something... only put "1" instead of "1.0.0" as it's shorter.